cairo/ethereum/crypto/hash.cairo

```diff
@@ -28,3 +30,43 @@ func keccak256{range_check_ptr, bitwise_ptr: BitwiseBuiltin*, keccak_ptr: Keccak
     tempvar hash = Hash32(value=value);
     return hash;
 }
+
+// @notice Computes the blake2s hash of a bytes object.
+// @dev `finalize_blake2s` must absolutely be called at the end of the program.
+func blake2s_bytes{range_check_ptr, blake2s_ptr: felt*}(buffer: Bytes) -> Hash32 {
+    alloc_locals;
+    let n_bytes = buffer.value.len;
+    let (dst: felt*) = alloc();
+    bytes_to_bytes4_little_endian(dst, n_bytes, buffer.value.data);
+    let (result) = blake2s(dst, n_bytes);
+    tempvar value = new Uint256(low=result.low, high=result.high);
+    tempvar hash = Hash32(value=value);
+    return hash;
+}
+
+// @notice Computes the hash of a bytes object using the given hash function.
+// @dev To avoid re-binding the arguments in the correct order, the `hash_function_name` must be the
```

Comment:
is this comment conveying correct information? the hash function name is not an implicit argument